client/components/navigation/Navbar.tsx

```diff
@@ -46,69 +53,172 @@ function Navbar() {
+              textDecoration="none"
+              borderRadius="18px"
+              href="/companies"
+            >
+              Companies
+            </Link>
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
+              cursor="pointer"
+              textDecoration="none"
+              borderRadius="18px"
+              href="/about"
+            >
+              About
+            </Link>
+            <Link textDecoration="none" href="/auth">
+              <ButtonBlack>Sign Up</ButtonBlack>
+            </Link>
+          </VStack>
+        )}
 
+        <HStack
+          spacing={{ base: 2, sm: 7 }}
+          display={{ sm: 'none', md: 'block' }}
+        >
+          <Link
+            color="black"
+            p="0.45rem"
+            m="0 0.25rem"
+            cursor="pointer"
+            textDecoration="none"
+            _hover={{ textDecoration: 'none', bgColor: '#e2e8f0' }}
+            _focus={{ textDecoration: 'none', border: 'none' }}
+            borderRadius="18px"
+            href="/gigs"
+          >
+            Gigs
+          </Link>
           <Link
+            color="black"
+            p="0.45rem"
+            m="0 0.25rem"
+            cursor="pointer"
             textDecoration="none"
-            href="/auth">
-              <ButtonBlack
-            >Sign Up</ButtonBlack>
+            borderRadius="18px"
+            _hover={{ textDecoration: 'none', bgColor: '#e2e8f0' }}
+            _focus={{ textDecoration: 'none', border: 'none' }}
+            href="/jobs"
+          >
+            Jobs
           </Link>
 
+          <Link
+            color="black"
+            p="0.45rem"
+            m="0 0.25rem"
+            cursor="pointer"
+            textDecoration="none"
+            _hover={{ textDecoration: 'none', bgColor: '#e2e8f0' }}
+            _focus={{ textDecoration: 'none', border: 'none' }}
+            borderRadius="18px"
+            href="/developers"
+          >
+            Devs
+          </Link>
+          <Link
+            color="black"
+            p="0.45rem"
+            m="0 0.25rem"
+            cursor="pointer"
+            textDecoration="none"
+            _hover={{ textDecoration: 'none', bgColor: '#e2e8f0' }}
+            _focus={{ textDecoration: 'none', border: 'none' }}
+            borderRadius="18px"
+            href="/companies"
+          >
+            Companies
+          </Link>
+          <Link
+            color="black"
+            p="0.45rem"
+            m="0 0.25rem"
+            cursor="pointer"
+            textDecoration="none"
+            _hover={{ textDecoration: 'none', bgColor: '#e2e8f0' }}
+            _focus={{ textDecoration: 'none', border: 'none' }}
+            borderRadius="18px"
+            href="/about"
+          >
+            About
+          </Link>
+          <Link textDecoration="none" href="/auth">
+            <ButtonBlack>Sign Up</ButtonBlack>
+          </Link>
```

Comment:
This is invalid HTML ("The element `button` must not appear as a decendant of the `a` element.") and it likely produces a result that isn't accessible.

Instead what we can do is wrap the button with `NextLink` (`import NextLink from 'next/link') and then render `Button as="a"` to make it an accessible link that looks like a button.

Here's a CodeSandbox example: https://codesandbox.io/s/intelligent-hill-tr608?file=/pages/index.tsx

Reply:
This was my mistake! Thanks for correcting!

Reply:
What's a mistake but a chance to learn something new? :D

The situation around links can be pretty confusing anyway. We should document that somewhere. Created #44 so I remember to do that 😆

Reply:
YES! 

By the way, I based the development of this Navbar off this: https://github.com/Developer-DAO/developerdao.com/blob/main/src/components/Nav/index.tsx